package.json

```
@@ -42,10 +52,11 @@
     "build:requested-delivery-date": "npm --prefix feature-libs/requested-delivery-date run build:schematics && nx build requested-delivery-date --configuration production",
     "build:schematics": "npm --prefix projects/schematics run build",
     "build:setup": "nx build setup --configuration production",
-    "build:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx run storefrontapp:server:production",
-    "build:ssr:opf": "env-cmd --no-override -e opf,b2c,$SPA_ENV nx run storefrontapp:server:production",
-    "build:ssr:ci": "env-cmd -e ci,b2c,$SPA_ENV nx run storefrontapp:server:production",
-    "build:ssr:local-http-backend": "env-cmd -e local-http,b2c,$SPA_ENV nx run storefrontapp:server:production",
+    "build:csr": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx run storefrontapp:build:noSsr",
+    "build:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV npm run build",
+    "build:ssr:opf": "env-cmd --no-override -e opf,b2c,$SPA_ENV npm run build",
+    "build:ssr:ci": "env-cmd -e ci,b2c,$SPA_ENV npm run build",
+    "build:ssr:local-http-backend": "env-cmd -e local-http,b2c,$SPA_ENV npm run build",
     "build:storefinder": "npm --prefix feature-libs/storefinder run build:schematics && nx build storefinder --configuration production",
```

[MEDIUM]
Let's not call transitively `npm run build`, because then we loose all the customized values of `env-cmd` (like `opf` or `local-http`) in favor of the `env-cmd` values defined in the `"build"` script of package.json.

In particular the the `npm run test:ssr` command fails on the `validate-ssr-build.ts` script, because there is no `USING_PROXY_BACKEND_MARKER` present in the dist /server folder. And it's because the command `build:ssr:local-http-backend` (which I've run before) didn't populate correctly the value `local-http` into `env-cmd` (because it was shadowed by `env-cmd` values defined inside the `"build"` script)

In other words, let me suggest the following changes:
```suggestion
 "build:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx run storefrontapp:build:production",
    "build:ssr:opf": "env-cmd --no-override -e opf,b2c,$SPA_ENV nx run storefrontapp:build:production",
    "build:ssr:ci": "env-cmd -e ci,b2c,$SPA_ENV nx run storefrontapp:build:production",
    "build:ssr:local-http-backend": "env-cmd -e local-http,b2c,$SPA_ENV nx run storefrontapp:build:production",
```

---

I assume that storefinder was accidentally removed. I added it back